cmd/template/cluster/provider/templates/azure/azure_machine_template.yaml.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: AzureMachineTemplate
+metadata:
+  labels:
+    "release.giantswarm.io/version": "{{ .Version }}"
+    "giantswarm.io/cluster": "{{ .Name }}"
+    "cluster.x-k8s.io/cluster-name": "{{ .Name }}"
+    "giantswarm.io/organization": "{{ .Owner }}"
+    "cluster.x-k8s.io/watch-filter": "capi"
+  name: {{ .Name }}-control-plane
+  namespace: {{ .Namespace }}
+spec:
+  template:
+    spec:
+      dataDisks:
+      - diskSizeGB: 256
+        lun: 0
+        nameSuffix: etcddisk
+      osDisk:
+        diskSizeGB: 128
+        osType: Linux
+      sshPublicKey: ""
```

Comment:
```suggestion
```

Reply:
Can't remove it. `apply` fails with validation error.